content/contents+ar.lr

```diff
@@ -148,14 +147,14 @@ size: lg
 ----
 is_block: yes
 #### text-block ####
-title: ### Members
+title: ### الأعضاء
 ----
-text: BeeWare is supported by users like you! Every little bit helps: `click here to contribute </contributing/membership/>`__.
+text: BeeWare مدعوم بمستخدمين مثلك ! أي  شيء سيساعد `اضغط هنا للمساهمة </contributing/membership/>`__.
 ----
-image: 
+image:
 ----
-image_alignment: 
+image_alignment:
 ---
-gutter_top: ### Many thanks to
+gutter_top: ### جزيل الشكر لـ
 ---
 gutter_bottom: ### [... and many more](/project/projects/)
```

Comment:
This is starting to look awesome :-)

How about `[... and many more]` text :-) ?